src/parquet/column/reader.h

```diff
@@ -58,6 +58,11 @@ class PARQUET_EXPORT ColumnReader {
 
   const ColumnDescriptor* descr() const { return descr_; }
 
+  int64_t GetRemainingInPage() {
```

Comment:
In our current implementation, for a given column `batch size`, values are not returned across DataPages.
Since, different columns (based on the type) can have different number of values in a `DataPage`, it becomes difficult to specify a `batch size` such that all columns return the same number of values.  i.e. a batch of rows is not returned.
The purpose of this is API is to query the number of remaining values in a Page for a given column. The client can now take the minimum of `GetRemainingInPage` in all the columns to specify a `batch size`.